std/assembly/index.d.ts

```diff
@@ -466,10 +473,15 @@ declare namespace i64 {
   export function div_s(left: i64, right: i64): i64;
   /** Computes the unsigned quotient of two 64-bit integers. */
   export function div_u(left: i64, right: i64): i64;
-  /** Return 1 two 64-bit inegers are equal to each other, 0 otherwise. */
+  /** Return 1 if two 64-bit integers are equal to each other, 0 otherwise. */
   export function eq(left: i64, right: i64): i32;
-  /** Return 0 two 64-bit inegers are equal to each other, 1 otherwise. */
+  /** Return 0 if two 64-bit integers are equal to each other, 1 otherwise. */
   export function ne(left: i64, right: i64): i32;
+  /** Compute the remaining between two 64-bit signed positive integers. */
+  export function rem_s(left: i64, right: i64): i64;
+  /** Compute the remaining between two 64-bit unsigned positive integers. */
+  export function rem_u(left: u64, right: u64): u64;
```

Comment:
```suggestion
  /** Computes the signed remainder of two 64-bit integers. */
  export function rem_s(left: i64, right: i64): i64;
  /** Computes the unsigned remainder of two 64-bit integers. */
  export function rem_u(left: u64, right: u64): u64;
```

Reply:
done in https://github.com/AssemblyScript/assemblyscript/pull/2306/commits/1130f0aac50bf058c79d91228484c4e872b53771